.vscode/settings.json

```diff
@@ -11,6 +11,6 @@
     "dist/": true
   },
   "editor.tabSize": 2,
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
In what way do you have to change the way you write code with this enabled?

Reply:
Prettier makes a lot of assumptions about ternaries, destructuring and spacing, and its kinda buggy at times too.

Reply:
But if you don’t disable the commit hook prettier still rewrites at commit time, so then this issue would remain. I would prefer we keep note of actual bugs and report/fix them instead.

Reply:
Yep, agreed that this option shouldn't be changed

Reply:
👍 will do.